### PR TITLE
Fix claiming issues in frontend

### PR DESCRIPTION
**Pull request by kschiffer**
#### Summary
This quickfix PR addresses some issues with device claiming in the Console and JS SDK.

#### Changes

- Fix claim info request when claiming device (JS SDK)
- Improve error messaging for failed device claims

#### Testing

Manual testing.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/device-onboarding-form/index.js [hunk 1]
```diff
@@ -113,6 +113,14 @@ const DeviceOnboardingForm = () => {
           delete mac_state.current_parameters
         }
 
+        if (values.authenticated_identifiers) {
+          cleanedValues.ids = {
+            ...cleanedValues.ids,
+            ...values.authenticated_identifiers,
+            device_id: values.target_device_id,
+          }
+        }
```
Comment (johanstokking) [inline pkg/webui/console/containers/device-onboarding-form/index.js]: This causes a regression in the manual device onboarding flow, causing `ids.dev_eui` to be set to `""` and `ids.authentication_code` to be set, which doesn't exist in `ids`.

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: pkg/webui/console/containers/device-onboarding-form/index.js (hunk 1 above)
Comment (johanstokking) [reply pkg/webui/console/containers/device-onboarding-form/index.js]: I think there are two issues here: 1) `values.authenticated_identifiers` seems to be always truthy as it's an object, but the values `dev_eui` and `authentication_code` can be empty and 2) this sets `authentication_code` from `authenticated_identifiers` in `ids` which is wrong, it should be at most the `dev_eui`.

Comment (johanstokking) [reply pkg/webui/console/containers/device-onboarding-form/index.js]:
```suggestion
        if (values.authenticated_identifiers.dev_eui) {
          cleanedValues.ids = {
            ...cleanedValues.ids,
            dev_eui: values.authenticated_identifiers.dev_eui,
            device_id: values.target_device_id,
          }
        }
```

**Review by adriansmares — APPROVED** [review]

File: sdk/js/src/service/devices/index.js [hunk 2]
```diff
@@ -521,12 +521,8 @@ class Devices {
               application_id: applicationId,
             },
           }
-      try {
```
Comment (kschiffer) [inline sdk/js/src/service/devices/index.js:524]: This try/catch and wrapping the error will make us loose the error shape we use in the Console to forward errors to the user.

**Review by ryaplots — APPROVED** [review]

**Pull request merged**